cypress.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = defineConfig({
             username: userName.toLowerCase(),
             email: 'test'+`${randomNumber}`+'@mail.com',
             password: '12345Qwert!',
```

Comment:
Consider using the faker library to generate a random email instead of concatenating a random number.